ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/api/services/grpc/GrpcCommandCompletionService.scala

```diff
@@ -21,26 +21,44 @@ import scalaz.syntax.tag._
 
 import scala.concurrent.Future
 
+object GrpcCommandCompletionService {
+
+  private[this] val completionStreamDefaultOffset =
+    LedgerOffset.of(LedgerOffset.Value.Boundary(LedgerOffset.LedgerBoundary.LEDGER_END))
+
+  private def fillInWithDefaults(request: CompletionStreamRequest): CompletionStreamRequest = {
```

Comment:
nitpicking: this method could live on the class itself, does not have to be "_static_". Also, you  can drop two `{`a and '}'s

Reply:
I usually favor _"static"_ methods unless I'm relying on a specific object's state. I can make it an instance method if you think it belongs there. Do you have a specific reason why you'd prefer it to be an instance method?

Reply:
Just so it's close to where it's used. I tend not put too many functions on `objects`, even if they are stateless, unless they need to have wider visibility. Putting them on the class instance has no runtime penalty and results in better code cohesion in my opinion. 

This is a matter of taste I think, don't change it if you don't agree.